target/linux/ath79/image/generic.mk

```diff
@@ -314,6 +342,28 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/belkin_f9j1108-v2
+  SOC := qca9558
+  DEVICE_VENDOR := Belkin
+  DEVICE_MODEL := AC1750 DB Wi-Fi (F9J1108v2)
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct
+  KERNEL_SIZE := 5120k
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0x50000
+  COMPILE := loader-$(1).bin loader-$(1).uImage
+  COMPILE/loader-$(1).bin := loader-okli-compile
+  COMPILE/loader-$(1).uImage := append-loader-okli $(1) | pad-to 64k | lzma | \
+	uImage lzma
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma -M 0x4f4b4c49
+  IMAGE/sysupgrade.bin := append-kernel | pad-to $$(KERNEL_SIZE) | append-rootfs | \
+	pad-rootfs | check-size
+  IMAGE/factory.bin := create-edimax-factory-bin F9J1108v1 BR-6679BAC \
+    $(KDIR)/loader-$(1).uImage $(KDIR_TMP)/openwrt-ath79-generic-$(1)-squashfs-sysupgrade.bin
+  IMAGES += sysupgrade.bin factory.bin
+  IMAGE_SIZE := 15936k
```

Comment:
14464k?

Reply:
Exactly right, only the firmware partition should be taken into account.